orchagent/bulker.h

```diff
@@ -11,6 +11,8 @@
 #include "logger.h"
 #include "sai_serialize.h"
 
+extern size_t gMaxBulkSize;
```

Comment:
>gMaxBulkSize [](http://example.com/codeflow?start=14&length=12)

Better to define in below classes, also its default value.

Reply:
Defined a member variable in the classes for the max bulk size and setting the value in the orch that creates the bulker. As the default is currently global, the default value is not added to the class.